src/app/config/scripts/read-ssm.js

```diff
@@ -0,0 +1,80 @@
+// @ts-check
+
+// Copyright The Linux Foundation and each contributor to CommunityBridge.
+// SPDX-License-Identifier: MIT
+const AWS = require('aws-sdk');
+
+/**
+ * @param {string[]} variables
+ * @param {string} stage
+ * @param {string} region
+ * @param {string} profile
+ * @returns {Promise<{ [key:string]: string}>}
+ */
+async function retrieveSSMValues(variables, stage, region, profile) {
+    const scopedVariables = variables.map((param) => {
+        return `cla-${param}-${stage}`;
```

Comment:
Notice here we add the prefix and suffix.  The prefix should match our SSM parameters - recommend we use: `vd`.  Change `cla` to `vd`.